test/basic.test.js

```diff
@@ -1,162 +1,86 @@
-import {
-	assert, html, fixture
-} from '@open-wc/testing';
+import { assert, html, fixture, nextFrame, oneEvent } from '@open-wc/testing';
 import '../cosmoz-datetime-input';
 
+const getDate = (el) => el.shadowRoot.querySelector('[type="date"]')?.value;
```

Comment:
Ideally we would not have to go beyond the public interface when testing the component. Could you adapt them so they rely solely on what is made public explicitly?